app/utils/nonsipp/summary/LoansCheckAnswersUtils.scala

```diff
@@ -83,7 +84,9 @@ trait LoansCheckAnswersUtils(schemeDateService: SchemeDateService)
       .flatten
 
   def summaryDataAsync(srn: Srn, index: Max5000, mode: Mode)(using
-    request: DataRequest[AnyContent]
+    request: DataRequest[AnyContent],
```

Comment:
@narendrapaduchuri would this cause any issues with the work you are doing right now?